code/style.css

```diff
@@ -2,6 +2,336 @@
+
+header {
+  position: relative;
+  width: 100%;
+  height: 50vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  font-family: 'Oswald', sans-serif;
+}
+
+header h1 {
+  text-align: center;
+  font-size: 40px;
+  z-index: 1;
+  color: #fff;
+  opacity: 0.9;
+  padding: 20px;
+  margin: 15px;
+}
+
+header h2 {
+  text-align: center;
+  font-size: 25px;
+  z-index: 1;
+  color: #fff;
+  opacity: 0.9;
+  margin-top: 20px;
+}
+
+video {
+  display: none;
+}
+
+picture {
+  position: absolute;
+  top:0;
+  left:0;
+  width: 100%;
+  object-fit: cover;
+}
+
+picture img {
+  height: 50vh;
+  width: 100%;
+}
+
+.main-container {
+  text-align: center;
+  background-color: #e1e1e1;
+  padding: 10px 0 30px 0;
+}
+
+.flex-box {
+  display: flex;
+  flex-direction: column;
+}
+
+/* FORM */
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  background-color: #fff;
+  width: 90%;
+  max-width: 400px;
+  border-radius: 5%;
+  margin-right: auto;
+  margin-left: auto;
+  padding: 40px 10px;
+}
+
+.form-group {
+  display: flex;
+  margin-bottom: 15px;
+  align-items: center;
 }
+
+.form-group label {
+  width: 150px;
+  margin-right: 5px;
+}
+
+::placeholder {
+  font-size: 12px;
+  font-family: Courier New, monospace;
+}
+
+.form-group input:valid {
+  border-color: #000; 
+}
+
+input:not(:placeholder-shown) {
+  font-weight: bold;
+}
+
+input[required]:placeholder-shown {
+  border-color: red;
+}
+
+.form-group input{ 
+  border: none;
+  border-bottom: 1px solid #555;  
+  background: transparent;
+  width:50%;
+  padding:8px 0 5px 0;
+  font-size:16px;
+  color:#000;
+}
+
+.radio-group {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 15px 10px;
+}
+
+.radio-group label {
+  margin-left: 0;
+  width: 150px;
+}
+
+.checkbox-group {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  margin-top: 10px;
+}
+
+.checkbox-group label {
+  margin-left: 0;
+  width: 150px;
+}
+
+.checkbox-group label .terms-box {
+  margin-left: 30px;
+}
+
+.form-button {
+  margin-top: 30px;
+}
+
+.submit-btn {
+  background-color: #555;
+  color: #fff;
+  padding: 8px 25px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 15px;
+  cursor: pointer;
+  border: 0;
+  border-radius: 20px;
+}
+
+.submit-btn:hover {
+  background-color: rgb(41, 40, 40);
+}
+
+/* Accordion */
+
+.accordion {
+  background-color: #fff;
+  color: #000;
+  width: 90%;
+  max-width: 400px;
+  border-radius: 2%;
+  margin: 20px auto;
+  padding: 5px 5px;
+  height: auto;
+}
+
+.question {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 15px;
+  background-color: rgb(238, 235, 235);
+}
+
+.question:hover {
+  background-color: rgb(197, 196, 196);
+  cursor: crosshair;
+}
+
+.question::after {
+  content: "\002B";
+  font-size: 20px;
+}
+
+.question.active::after {
+  content: "\2212";
+}
```

Comment:
To create a smoother transition try to use the Inspector tool in the browser and check what's happening when you click the accordion to target the right class. Maybe some good info here too:
http://css3.bradshawenterprises.com/accordions/